central/detection/lifecycle/manager_impl.go

```diff
@@ -168,33 +170,63 @@ func (m *managerImpl) flushBaselineQueue() {
 
 		// Grab the first deployment to baseline.
 		// NOTE:  This is the only place from which Pull is called.
-		deployment := m.deploymentObservationQueue.Pull()
+		deploymentObs := m.deploymentObservationQueue.Pull()
```

Comment:
nit: If you are going to name it, `Obs` is not clear.  I probably would have left this one as deployment and named the other one `fullDeployment` or something.